project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/services/MinimizationPsiManagerService.kt

```diff
@@ -50,6 +60,13 @@ private typealias NodesAndEdges = Pair<List<PsiStubDDItem>, List<PsiIJEdge>>
 class MinimizationPsiManagerService {
     private val logger = KotlinLogging.logger {}
 
+    private val KtElement.selfOrConstructorIfFunctionOrClass: KtFunction?
```

Comment:
What about secondary constructors?

Reply:
Regarding this extension property, it needed to get the primary constructor, since it is the only one that could declare properties

Reply:
Got it